server/ot_rpc_test.go

```diff
@@ -0,0 +1,98 @@
+package server
+
+import (
+	"bytes"
+	"crypto/tls"
+	"io"
+	"io/ioutil"
+	"math/rand"
+	"mime"
+	"mime/multipart"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+
+	"github.com/livepeer/go-livepeer/common"
+	"github.com/livepeer/go-livepeer/core"
+	"github.com/livepeer/go-livepeer/eth"
+	"github.com/livepeer/go-livepeer/net"
+	"github.com/livepeer/lpms/ffmpeg"
+	"github.com/stretchr/testify/assert"
+)
+
+type stubTranscoder struct {
+	called int
+	fname  string
+}
+
+var testRemoteTranscoderResults = [][]byte{[]byte("body1"), []byte("body2")}
+
+func (st *stubTranscoder) Transcode(fname string, profiles []ffmpeg.VideoProfile) ([][]byte, error) {
+	st.called++
+	st.fname = fname
+	return testRemoteTranscoderResults, nil
+}
+
+func TestRemoteTranscoder(t *testing.T) {
```

Comment:
Can some comments be added throughout here describing what is being tested? It's kind of hard to tell by just scanning through the code.